Client/Clique/Common/Functions/Functions.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Functions.swift
+//  Clique
+//
+//  Created by Infinum on 09.12.2022..
+//
+
+import UIKit
+import NVActivityIndicatorView
+
+enum Functions {
+    
+    enum Alerts {
+        
+        static func alert(fwdMessage: String, viewController: UIViewController) {
+            let alertController = UIAlertController(title: "", message: fwdMessage, preferredStyle: .alert)
+            let defaultAction = UIAlertAction(title: "OK", style: .cancel, handler: nil)
```

Comment:
If you have a constant for e.g. `fwdMessage`, please make a constant for "OK" as well.

Btw. `fwdMessage` should be renamed a bit, don't know what `fwd` means in this context.